.github/scripts/build_publish_nightly_docker.sh

```diff
@@ -3,13 +3,13 @@
 set -xeuo pipefail
 
 PYTORCH_DOCKER_TAG=$(git describe --tags --always)-devel
+CUDA_VERSION=11.1.1
```

Comment:
The only negative here is that since we don't actually have CUDA 11.1 nightly builds yet this workflow will fail until then. I'd suggest waiting on merging this until https://github.com/pytorch/pytorch/pull/52151 is merged